src/mini_allocator.c

```diff
@@ -614,10 +666,12 @@ mini_alloc(mini_allocator *mini,
  *-----------------------------------------------------------------------------
  * mini_release --
  *
- *      Called to finalize the mini_allocator. After calling, no more
- *      allocations can be made, but the mini_allocator linked list containing
- *      the extents allocated and their metadata can be accessed by functions
- *      using its meta_head.
+ *      Called to finalize the mini_allocator. As the mini-allocator always
+ *      pre-allocates an extent, 'release' here means to deallocate this
+ *      pre-allocated extent for each active batch. After calling release, no
+ *      more allocations can be made, but the mini_allocator's linked list
+ *      containing the extents allocated and their metadata can be accessed by
+ *      functions using its meta_head.
```

Comment:
I tried to infer the workings from the code and inscribed in this comment. Please review for accuracy and suggest changes.